doc/Extensions/Distributed-Poller.md

```diff
@@ -56,9 +56,25 @@ Central storage should be provided so all RRD files can be read from and written
 
 For this example, we are running RRDCached to allow all pollers and web/api servers to read/write to the rrd files with the rrd directory also exported by NFS for simple access and maintenance.
 
+Install RRD tool from source on the pollers and your main system to make sure you have RRDTool >=1.5.5. RRDTool 1.7 is the lastest version.
+
+On your main server you need to run rrdcached as a service listening on a TCP port. Run the following command as a deamon:
+
+```
+/usr/bin/rrdcached -l :42217 -w 1800 -z 1800 -f 3600 -s librenms -U librenms -G librenms -B -R -j /var/tmp -t 4 -F -b /opt/librenms/rrd/
```

Comment:
Thanks for this, this needs extending out to cover the systemd files as people won't know what to do with the below.